hibernate-community-dialects/src/main/java/org/hibernate/community/dialect/sequence/LegacyDB2SequenceSupport.java

```diff
@@ -4,9 +4,10 @@
  * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
  * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
  */
-package org.hibernate.dialect.sequence;
+package org.hibernate.community.dialect.sequence;
```

Comment:
Please undo the move of this class and simply annotate the class with `@Deprecated`, because the change of the package name is not backwards compatible.